src/bp-messages/classes/class-bp-messages-thread.php

```diff
@@ -1655,16 +1655,6 @@ public static function get( $args ) {
 			$sql['pagination'] = $wpdb->prepare( 'LIMIT %d, %d', intval( ( $r['page'] - 1 ) * $r['per_page'] ), intval( $r['per_page'] ) );
 		}
 
-		/**
-		 * Filters the Where SQL statement.
-		 *
-		 * @since BuddyBoss 1.5.4
-		 *
-		 * @param array $r                Array of parsed arguments for the get method.
-		 * @param array $where_conditions Where conditions SQL statement.
-		 */
-		$where_conditions = apply_filters( 'bp_recipients_recipient_get_where_conditions', $where_conditions, $r );
-
```

Comment:
why removed this filter its introduce from 1.5.4

Reply:
My apologies for this. I have re-added this apply_filter now. @KartikSuthar